### PR TITLE
adding og:see_also to public meta tags using the library recommendations

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

**Comment by eishay** [comment]
@joshm1

**Comment by joshm1** [comment]
lgtm!

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PageMetaTagsCommander.scala [hunk 1]
```diff
@@ -0,0 +1,112 @@
+package com.keepit.commanders
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PageMetaTagsCommander.scala:1]: Nice, thanks for splitting it out.

**Comment by andrewconner** [comment]
Looks good.

**Pull request merged**